contracts/acl/ACLSyntaxSugar.sol

```diff
@@ -85,18 +87,34 @@ contract ACLSyntaxSugar {
 }
 
 
-contract ACLHelpers {
-    function decodeParamOp(uint256 _x) internal pure returns (uint8 b) {
+contract ACLHelpers is ACLParams {
+
```

Comment:
Why are we adding 2 newlines here? Not a big deal, but we don't usually do that.